test/integration/api/event.spec.js

```diff
@@ -118,6 +118,32 @@ test('get events history with date filter', async (t) => {
   })
 })
 
+test('fails to get events history with same date boundary type filters', async (t) => {
+  const authorizationHeaders = await getAccessTokenHeaders({
+    t,
+    permissions: [
+      'event:stats:all',
+    ]
+  })
+
+  const now = new Date().toISOString()
+  const date = computeDate(now, '-10d')
+
+  const { body: error1 } = await request(t.context.serverUrl)
+    .get(`/events/history?groupBy=day&createdDate[gt]=${date}&createdDate[gte]=${date}`)
+    .set(authorizationHeaders)
+    .expect(400)
+
+  t.true(error1.message.includes('optional exclusive peers'))
+
+  const { body: error2 } = await request(t.context.serverUrl)
+    .get(`/events/history?groupBy=day&createdDate[lt]=${date}&createdDate[lte]=${date}`)
+    .set(authorizationHeaders)
+    .expect(400)
+
+  t.true(error2.message.includes('optional exclusive peers'))
```

Comment:
Can you share the full Joi message here as a Github comment for reference?

Reply:
Sure. Here's the error message:
`"createdDate" contains a conflict between optional exclusive peers [gt, gte]`